.goreleaser.yml

```diff
@@ -20,7 +20,7 @@ builds:
       - ppc64le
```

Comment:
Hi @OchiengEd 

That seems great now.
Could you please just squash the commits so that we can ensure a nice log for it when we run `git log` after it get merged? It does not seems that requires 9/10 commits 

Also, I think you can move forward with this one .. it does not need to be a draft anymore it seems good to fly 🚀

Reply:
Sounds good. Working on it

Reply:
done